plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -307,8 +307,8 @@ export class GraphQLInstrumentation extends InstrumentationBase {
         schema: graphqlTypes.GraphQLSchema,
         documentAST: graphqlTypes.DocumentNode,
         rules?: ReadonlyArray<graphqlTypes.ValidationRule>,
-        typeInfo?: graphqlTypes.TypeInfo,
-        options?: { maxErrors?: number }
+        options?: { maxErrors?: number },
+        typeInfo?: graphqlTypes.TypeInfo
```

Comment:
`graphql` v16 flips around these two parameters 🤯 . This has no effect at runtime when using v15 Vs v16 because they both pass whatever they receive through, but needs to be changed in the case we want to compile against v16.